org-code-javabuilder/theater/src/main/java/org/code/theater/Theater.java

```diff
@@ -0,0 +1,181 @@
+package org.code.theater;
+
+public enum Instrument {
+   PIANO, BASS
+}
+
+public static class Theater {
+   /**
+    * Plays the array of samples provided.
+    * 
+    * @param sound an array of samples to play.
+    */
+   public static void playSound(double[] sound) {
+   }
+
+   /**
+    * Plays the sound referenced by the file name.
+    * 
+    * @param filename an array of samples to play.
+    * @throws SoundException if the file can't be found in the project.
```

Comment:
would it make more sense to throw a `FileNotFoundException` instead of a `SoundException`?

Reply:
Oh, yes, that's much better.